.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # testing
 /coverage
+cypress/videos/
```

Comment:
Running cypress locally also added some files to cypress/fixtures/. I think that also should be gitignored